### ISSUE TITLE
compiled error after upgraded to crystal 0.20.0

### ISSUE THREAD

**Issue by dylandy** [greyblake/crystal-icr#35]
MemoryIO  has renamed to IO::Memory in this version which leads to the following error

```crystal
/usr/bin/crystal build --release -o bin/icr src/icr/cli.cr 
Error in src/icr/cli.cr:2: while requiring "../icr"

require "../icr"

in src/icr.cr:3: while requiring "io/memory_io": can't find file 'io/memory_io'

require "io/memory_io"
^

make: *** [build] Error 1

```

**Comment by dylandy** [comment]
this issue should be fixed in pull request #34

**Comment by greyblake** [comment]
I've merged that! Thanks everyon!
